debian/changelog

```diff
@@ -1,3 +1,10 @@
+cocaine-core (0.12.0.2) unstable; urgency=low
```

Comment:
We already have 0.12.0.1 in https://github.com/cocaine/cocaine-core/pull/165/files#diff-f758558c8bab6e325c78465f6a7fa64bR132